plugins/testing/skills/mutation-generator/generate_mutations.py

```
@@ -0,0 +1,443 @@
+                print(f"   Created {i}/{len(self.mutations)} mutants...")
+    
+    def _apply_mutation_to_file(self, file_path: Path, mutation: Dict[str, Any]):
+        """Apply mutation to a specific file."""
+        with open(file_path, 'r') as f:
+            lines = f.readlines()
+        
+        target_line = mutation['line'] - 1
+        
+        if target_line < len(lines):
+            # Replace the line
+            original_content = lines[target_line].strip()
```

Ruff (0.14.10)

Local variable `original_content` is assigned to but never used

Remove assignment to unused variable `original_content`

(F841)